### ISSUE TITLE
Access Page ContentItem from liquid widget

### ISSUE THREAD

**Issue by ns8482e** [OrchardCMS/OrchardCore#6095]
How do I access ContentItem of  Page from liquid widget? when I use Model.ContentItem it refers to itself.

**Comment by deanmarcussen** [comment]
Do you mean you have a `Page` with a `FlowPart`, and you want to access a value from the `Page` when you are in your `Widget` template?

If so, you would need to pass the value from the `Page` to the `Widgets` in the shape

**Comment by ns8482e** [comment]
@deanmarcussen yes Liquid widget is on FlowPart. In Page content item, I have some fields that i need to access within liquid widget.

**Comment by deanmarcussen** [comment]
Can you use something like the `shape_add_properties` in a `FlowPart` template for that page to pass those field values into the widgets

**Comment by sebastienros** [comment]
I can't find a way to do it for now. The FlowPart shape has access to the Widget shapes, but the LiquidPart has only access to the Widget content item.

FlowPart shape can add something to the widget shape, but it would need to add it to the widget content item. Or we could use an ambient content item, like the aspects but at the requests level, like a HttpContext `Feature`. The content item controller could define such a feature.
